pkg/controllers/metrics/pods.go

```diff
@@ -0,0 +1,72 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"strings"
+
+	"github.com/awslabs/karpenter/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/multierr"
+	v1 "k8s.io/api/core/v1"
```

Comment:
unneeded alias

Reply:
`goimports` re-adds the alias if I try to remove it.  Found this [issue](https://github.com/golang/go/issues/29041) pertaining to `goimports` and packages named "v1".